embedded/scripts/env.sh

```diff
@@ -0,0 +1,2 @@
+export RUSTFLAGS="-C link-arg=-Tlink.x"
```

Comment:
maybe just a personal preference but isn't better a `.cargo/config` instead of `env.sh`?

Reply:
I find it easier to try out different configurations this way.  Also, it matches what was done in bitcoin_hashes and rust-bech32, so if we do want to change the approach, we should coordinate.

Reply:
Oh ok, I was remembering wrong for bitcoin_hashes and rust-bech32